File: Reports/Daniil_Pozdnyakov/lab4/src/task3/WorkingPerson.cs
namespace lab4._3._5
{
    public class WorkingPerson : Person, Worker
    {
        private bool isWorking = false;

        public WorkingPerson(int Age, string Name) : base(Age, Name)
        {

        }
        public void Work()
        {
            isWorking = true;
        }

        public bool IsWorking()
        {
            return isWorking;
        }

        public void f_stop_working()
        {
            isWorking = false;
        }
    }
}

File: reports/Ignatyuk/Lab5/src/Task3/Model/src/model/WorkingPerson.java
package model;

public abstract class WorkingPerson extends Person implements Worker {
    private Boolean m_IsWorking = Boolean.FALSE;

    public WorkingPerson(final Integer c_Age, final String c_Name) {
        super(c_Age, c_Name);
    }

    public final void f_work() {
        this.m_IsWorking = Boolean.TRUE;
    }

    public final Boolean f_is_working() {
        return this.m_IsWorking;
    }

    public final void f_stop_working() {
        this.m_IsWorking = Boolean.FALSE;
    }
}
